app/views/posts/index.html.erb

```diff
@@ -1,31 +1,31 @@
 <h2>Here is the post for given User</h2>
+  <section class = 'user'>
+    <figure>
+      <image class = 'user-image' src = '<%= @user.photo %>' alt = '<%= @user.name %>'>
+    </figure>
+    <div class='user-info'>
+      <h2>User: <%= @user.name %> </h2>
+      <p>Number of posts: <%= @user.post_counter || 0 %> </p>
```

Comment:
Same here. The post counter is not working as expected. It is showing 0 even though there is one post. Kindly fix this issue. 
![image](https://github.com/LarryIVC/larrys-blog/assets/33268927/621b8490-9597-4281-b764-8ae52865abba)

Reply:
Done